### ISSUE TITLE
subclassing in a different module

### ISSUE THREAD

**Issue by amanica** [ipod825/OOPMacro.jl#1]
I figured out that to be able to subclass something in a different module you need to expose the class eg. `MyClass` and `AbstractClass`.

maybe we should mention it in the readme OR
Maybe we can add another macro to do it for you eg. 
`@export_class MyClass`

if that sounds good I can make a pull request

**Comment by ipod825** [comment]
Yeah, that sounds like a good uses case. But I think this package is outdated now since julia 0.6 is now out...

**Comment by amanica** [comment]
It still works great in 0.6.4
I can't imagine using Julia for a big project without this plugin - I think in objects :smiley:

**Comment by ipod825** [comment]
Though I am not using Julia now, you are still welcomed to send a pull request.
